rust/lib/src/lib.rs

```diff
@@ -32,31 +29,33 @@ fn construct_uri_load_config(
     data_dir: String,
     chain_hint: String,
     monitor_mempool: bool,
-) -> Result<(zingoconfig::ZingoConfig, http::Uri), String> {
+) -> Result<(ZingoConfig, http::Uri), String> {
     let lightwalletd_uri = construct_lightwalletd_uri(Some(uri));
 
-    let chain_hint_str: &str = chain_hint.as_str();
-
-    use zingoconfig::ChainType::*;
-    let chaintype = match chain_hint_str {
-        "main" => Mainnet,
-        "test" => Testnet,
-        "regtest" => Regtest,
+    let chaintype = match chain_hint.as_str() {
+        "main" => ChainType::Mainnet,
+        "test" => ChainType::Testnet,
+        "regtest" => ChainType::Regtest,
         _ => return Err("Error: Not a valid chain hint!".to_string()),
     };
+    let regtest_network: Option<RegtestNetwork> = match chaintype {
+        ChainType::Regtest => Some(RegtestNetwork::all_upgrades_active()),
+        _ => None,
+    };
     let mut config = match zingolib::load_clientconfig(
         lightwalletd_uri.clone(),
         None,
         chaintype,
         monitor_mempool,
+        regtest_network,
```

Comment:
I can't find this new parameter in `zingolib` yet, I don't know if there are some PR waiting to merge in zingolib with this new spec... obviously the build in CI is failing now...

Reply:
Ah! `zingoconfig::RegtestNetwork` is the new type! You can call `RegtestNetwork::new(overwinter, sapling, blossom, canopy, heartwood, orchard)` to specify network upgrade heights, or `RegtestNetwork::all_upgrades_active()` to just set them all to 1.

Reply:
sorry i havent updated this to most recent changes to zingolib, i will do that now!
currently zingo-mobile does not have the ability to set activation heights to higher than 1 so for now we will be using RegtestNetwork::all_upgrades_active() for all new zingoconfigs. i will work on implementing this soon but it will require some possibly major changes that i dont want to merge just before release